docs/data/base/components/input/OTPInput.js

```diff
@@ -0,0 +1,204 @@
+import * as React from 'react';
+import { Input as BaseInput } from '@mui/base/Input';
+import { Box, styled } from '@mui/system';
+
+function OTP({ seperator, inputCount, otp, setOtp }) {
+  const inputRefs = React.useRef(new Array(inputCount).fill(null));
+
+  const focusInput = (targetIndex) => {
+    const targetInput = inputRefs.current[targetIndex];
+    targetInput.focus();
+  };
+
+  const selectInput = (targetIndex) => {
+    const targetInput = inputRefs.current[targetIndex];
+    targetInput.select();
+  };
+
+  const handleKeyDown = (event, currentIndex) => {
+    switch (event.key) {
+      case 'ArrowLeft':
```

Comment:
Please handle ArrowUp the same way as ArrowLeft and ArrowDown the same way as ArrowRight

Reply:
Both `ArrowUp` and `ArrowLeft` does nothing (not even deselection of input) in mentioned benchmarks . so i reflected same behaviour in https://github.com/mui/material-ui/pull/40539/commits/bb2f3a83364c06f2e173196191ea45354b381bef